src/pages/workspaces/workspace/Data.js

```diff
@@ -354,7 +354,7 @@ const DataTypeSection = ({ title, titleExtras, error, retryFunction, children })
       tooltip: 'Error loading, click to retry.'
     }, [icon('sync', { size: 18 })]) : titleExtras
   ]),
-  children?.length && div({
+  !!children?.length && div({
```

Comment:
without this, the Reference Data children displays as "0" (literally the string "0") … let's make sure this resolves to a boolean.

Reply:
@davidangb can you explain how "!!" converts the string to a boolean? I haven't seen this syntax before (nor have I seen "children?.length"). Thanks!

Reply:
The `!!` is a terse (but common) way of saying "cast to Boolean". The first `!` casts truthy/falsy values to Boolean, and then returns the `NOT` of that value; the second `!` adds another `NOT`, so the two `NOT`s cancel each other out and the end result is the original value, but cast to Boolean. Here's a reference:
https://betterprogramming.pub/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054

The `.?` is a null-safe way of referencing values. Here's a reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
@davidangb Thanks for the references! The optional chaining is really cool-- I've been stuck writing "old JavaScript" for various reasons, and it's exciting to actually start learning about these newer features.